README.md

```diff
@@ -8,30 +8,43 @@ The **Rep**roducible **Se**lf **P**ublishing toolkit demonstrates how to reuse y
 
 In these examples, analysis does not have to be initiated manually, and output elements do not have to be copied, manually scaled, styled, or otherwise manipulated.
 Data analysis is kept in one and only one place, and configurable styling is applied programmatically at the document or output element level.
-Data *and* code dependencies are monitored for update via checksums, and are either provided or specified, so that both the toolkit in its present incarnation - as well as your own derivatives - can be reproduced locally and autonomously by your colleagues, reviewers, students, and everybody else. 
-As no binaries are tracked, publications built analogously to our examples are excellently suited for collaborative editing and version tracking, e.g. via Git. 
+Data *and* code dependencies are monitored for update via checksums, and are either provided or specified, so that both the toolkit in its present incarnation - as well as your own derivatives - can be reproduced locally and autonomously by your colleagues, reviewers, students, and everybody else.
+As no binaries are tracked, publications built analogously to our examples are excellently suited for collaborative editing and version tracking, e.g. via Git.
```

Comment:
With my configuration of emacs, saving automatically kills trailing whitespace, This is the only difference in this diff.